# DP: Makes it so sparc32 remains binary compatible with older releases

--- gcc/crtstuff.c.orig	Mon Apr 26 20:11:08 1999
+++ gcc/crtstuff.c	Mon Apr 26 20:12:28 1999
@@ -89,11 +89,12 @@
    
 /* References to __register_frame_info and __deregister_frame_info should
    be weak in this file if at all possible.  */
-extern void __register_frame_info (void *, struct object *)
-				  TARGET_ATTRIBUTE_WEAK;
+extern void __register_frame_info (void *, struct object *);
 
-extern void *__deregister_frame_info (void *)
-				     TARGET_ATTRIBUTE_WEAK;
+extern void *__deregister_frame_info (void *);
+/* DEBIAN/SPARC - Changed the references to the
+ * previous for binary compatibility reasons
+ */
 
 /* Provide default definitions for the pseudo-ops used to switch to the
    .ctors and .dtors sections.
